wallet-webapps-admin/src/main/webapp/vue-app/components/admin/wallets/AdminWallet.vue

```diff
@@ -143,14 +134,11 @@ export default {
     tokenBalance() {
       return (this.adminWallet && this.adminWallet.tokenBalance) || 0;
     },
-    tokenBalanceLabel() {
-      return `${this.walletUtils.toFixed(this.tokenBalance)} ${this.tokenSymbol}`;
-    },
     etherBalance() {
       return (this.adminWallet && this.adminWallet.etherBalance) || 0;
     },
     etherBalanceLabel() {
-      return `${this.walletUtils.toFixed(this.etherBalance)} eth`;
+      return `${this.walletUtils.toFixed(this.etherBalance)} E`;
```

Comment:
Colul dyou use Ether symble here ?

Reply:
as I know we don't have a symbol for Ether! they used to use "eth".